packages/components/src/components/wallet-tile/wallet-tile.scss

```diff
@@ -0,0 +1,61 @@
+.wallet-tile {
+    display: flex;
+    align-items: center;
+    justify-content: flex-start;
+    background-color: $color-white;
+    padding: 1rem 2rem;
+    border-radius: $BORDER_RADIUS;
+    flex-grow: 1;
+
+    &--mobile {
+        flex-direction: column;
+        align-items: flex-start;
+
+        .wallet-tile {
+            &__header {
+                display: flex;
+                justify-content: flex-start;
+                align-items: flex-end;
+                margin-bottom: 0.4rem;
+            }
+
+            &__icon {
+                margin-right: 0.4rem;
+            }
+        }
+    }
+
+    &--hover {
+        &:hover {
+            background-color: $color-grey-4;
+        }
+    }
+
+    &--active {
+        background-color: $color-grey-5;
+    }
+
+    &__icon {
+        margin-right: 1.6rem;
+        min-width: 4rem;
+    }
+
+    &__content {
+        display: flex;
+        justify-content: space-between;
+        align-items: center;
+        flex-grow: 1;
+    }
+
+    &__jurisdiction {
+        display: flex;
+        flex-direction: row;
+        justify-content: flex-end;
+        align-items: flex-start;
+        gap: 0.4rem;
+        border: 1px solid $color-black-1;
+        border-radius: $BORDER_RADIUS / 2;
+        padding: 0.2rem 0.4rem;
```

Comment:
@hamid-deriv according to the figma files, the padding of the badge must be `padding: 0 4px` which is the same used for the badge component.